devel/clavus/Portfile

```diff
@@ -4,9 +4,9 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cargo  1.0
 
-github.setup        peter-mbx clavus 0.0.1
+github.setup        peter-mbx clavus 1.1.0
 github.tarball_from archive
-revision            0
+revision            1
```

Comment:
Since the version went up, there is no reason to increment `revision`.

`revision` is meant to be incremented only when changes need to be made to the port, but the version isn't changing.